maths/fibonacci.py

```diff
@@ -80,6 +80,14 @@ def fib_recursive(n: int) -> list[int]:
     def fib_recursive_term(i: int) -> int:
         """
         Calculates the i-th (0-indexed) Fibonacci number using recursion
+        
+        NOTE: The current recursion depth limit is 1000 (thus n > 1000 won't work) and it will throw an error if it's exceeded to prevent infinite recursion / stack overflow.
+              The limit can be changed with the sys module, but setting it too high might evoke a crash:
+          
+              import sys
+              sys.setrecursionlimit(limit)
+          
+              A better solution is to rewrite the code without recursion e.g. with loops, like the iterative method.
```

Comment:
Two points:

1. may not be a good idea to exceed 100 symbols/line.
2. won't this chunk raise a `sphinx` warning if we try generating docs?

Reply:
I'm sorry but may you explain what a sphinx warning is? Thank you

Reply:
Documentation generator ([link](https://www.sphinx-doc.org/en/master/usage/quickstart.html)).

OK, we can skip the 2nd point, just ensure each line is not longer than 100 symbols

Reply:
Ah ok I'll try shorten those texts

Reply:
Or try moving text to the next line

Reply:
> Or try moving text to the next line

Good idea 😅

Reply:
Now, it does not exceed. I hope docs are only for developers to see, so now it's pretty fine regarding length